sycl/test/fpga_tests/buffer_location_codegen.cpp

```diff
@@ -0,0 +1,47 @@
+// RUN: %clangxx -fsycl -c -fsycl-device-only -S -emit-llvm %s -o - | FileCheck %s
```

Comment:
Please move this test to `check_device_code` directory.

Reply:
I prefer to handle this comment later

Reply:
Sure